src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -293,24 +308,67 @@
-        catchError((err) => {
-          this.lookupFailed = true;
-          return throwError(err);
-        }),
-        finalize(() => from(this.loaderService.hideLoader()))
-      )
-      .subscribe((selection) => {
-        this.modalController.dismiss({
-          selection,
-        });
+    setupPermissionDeniedPopover(): Promise<HTMLIonPopoverElement> {
+      let title = 'Location permission';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'title' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)